.github/workflows/ci.yml

```diff
@@ -27,10 +29,15 @@ jobs:
       -
         name: Docker meta
         id: docker_meta
-        uses: crazy-max/ghaction-docker-meta@v1
+        uses: crazy-max/ghaction-docker-meta@v3
         with:
           images: ${{ steps.reponame.outputs.DOCKER_REPO }} # list of Docker images to use as base name for tags
-          tag-sha: true # add git short SHA as Docker tag
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{major}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{version}}
+            type=sha
```

Comment:
As i understand `{{version}}` will match a full semver version and the git tag should be v1.2.3 which will push tag 1.2.3?

Reply:
Is `latest` by default always pushed even when only creating tags? i have some conditional below to push latest when master changes.

Reply:
ad semver version: Git tag can be `1.2.3` or `v1.2.3` , both will produce docker tags 
- `1.2.3` 
- `1.2`
- `1`
- sha
ad latest: I've now changed the behaviour with
```
flavor: |
  latest=false
```
so latest is now generated only for master with existing action

Reply:
👍